projects/skia/build.sh

```diff
@@ -57,19 +57,6 @@ export LDFLAGS_ARR=`echo $LDFLAGS | sed -e "s/\s/\",\"/g"`
 
 # Even though GPU is "enabled" for all these builds, none really
 # uses the gpu except for api_mock_gpu_canvas
-$SRC/depot_tools/gn gen out/Fuzz_mem_constraints\
```

Comment:
I'm sorry, I misunderstood.  We do need Fuzz_mem_constrains and Fuzz to be different builds.  As it stands, none of the fuzzers will now use the patch and I think we still need that.

Reply:
>As it stands, none of the fuzzers will now use the patch and I think we still need that.

I agree we need it. 
But I'm unclear why we need a separate output directory? As long as they use the same $SRC, I don't see how one can be using the patch but the other isn't. I think this was true before my change. But please let me know if I misunderstood something.